### PR TITLE
Add missing specs for plugins

### PR DESCRIPTION
**Pull request by stbenjam**
Doesn't this feel redundant?

### CODE REVIEW

File: spec/classes/foreman_plugin_abrt_spec.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'spec_helper'
+
+describe 'foreman::plugin::abrt' do
+  on_supported_os.each do |_os, facts|
```
Comment (ekohl) [inline spec/classes/foreman_plugin_abrt_spec.rb]: Does this set the facts correctly?

Comment (stbenjam) [inline spec/classes/foreman_plugin_abrt_spec.rb]: Ah sorry, no :-) My template was wrong, i need to do the let facts I guess.

**Comment by ekohl** [comment]
I believe there should be a test for every class, but I would agree that testing on every supported OS is a bit redundant.

**Comment by stbenjam** [comment]
It also looks like you all are hitting a lot of timeouts on the container infrastructure at Travis. :-(  This is going to add a bajillion more duplicate tests.

**Comment by ekohl** [comment]
Travis is the reason that I'd say running on some default OS is sufficient.

**Comment by stbenjam** [comment]
I had that in the other PR and @domcleal asked to do `.each`

**Comment by domcleal** [comment]
Ah okay, ignore me, @ekohl's right then.

**Comment by ekohl** [comment]
I wonder why the ruby 2.1.5 tests are failing.

**Comment by mmoll** [comment]
theforeman/puppet-foreman_proxy#188 is also failing all the way now with e.g. FQDN not set in the expections

**Comment by mmoll** [comment]
tests are OK again :)

**Comment by ekohl** [comment]
:+1:

**Comment by mmoll** [comment]
merged, thanks @stbenjam!

**Pull request closed without merging**